Source/EmbeddedBoundary/WarpXInitEB.cpp

```diff
@@ -241,9 +236,72 @@ WarpX::ScaleAreas() {
                 full_area = cell_size[0]*cell_size[1];
             }
             auto const &face_areas_dim = m_face_areas[maxLevel()][idim]->array(mfi);
-            amrex::LoopOnCpu(amrex::convert(box, amrex::Box(face_areas_dim).ixType()),
-                             [=](int i, int j, int k) {
+            auto const &mod_areas_dim = m_area_mod[maxLevel()][idim]->array(mfi);
+
+            amrex::ParallelFor(box, [=](int i, int j, int k) {
```

Comment:
Shall we add this, too? Cc: @WeiqunZhang 
```suggestion
            amrex::ParallelFor(box, [=] AMREX_GPU_DEVICE (int i, int j, int k) {
```